plugin.video.npogemist/resources/lib/pypo.py

```diff
@@ -0,0 +1,281 @@
+# -*- coding: utf-8 -*-
+"""Simplest possible language utility class.
+
+When runs as main, this will create the files strings.py, local_strings.py and
+kodi_strings.py, which are used to help localization, but using
+the Kodi utility functions. For each localized string, a variable will be created
+with a sensible name, which you can use to look up translated strings.
+
+Syntax:
+pypo.py [<pot_path>] [<kodi_resource_path>]
+
+where kodi resource path is a directory with standard kodi resources, e.g.
+%USERPROFILE%\\AppData\\Local\\Packages\\XBMCFoundation.Kodi_4n2hpmxwrvr6p\\LocalCache\\Roaming\\Kodi\\addons\\resource.language.nl_nl\\resources
+
+If you specify a kodi resource path, kodi_strings.py will be generated
+as well for standard strings. You need to do this once.
+If you don't specify it, only strings.py and local_strings.py will be generated.
+
+When included as a module, lang.get_string(language.string.resourceid) wil
+return a localized string, either from the addon resources, or from
+the official Kodi resources (tried in that order.) It will use
+kodi_strings and local_strings mentioned above to look up the variable used,
+so your IDE can be helpful.
+
+So you can use this class like this:
+- Use the following in your plugin:
+      po = Po(addon, xbmc)
+      s = po.get_string
+- Use resources in the form s(pypo.strings.mystring)
+- add entries in .po files
+- If you add new resources in your strings.po, re-run pypo.py and check the pypo.strings.variables used
+"""
+import keyword
+import string
+import sys
+import os.path
+from codecs import open
+
+# If used as a module, unconditionally require strings to exist.
+if __name__ != "__main__":
+    import local_strings
+    import kodi_strings
+    import strings
+
+
+class Po(object):
+    def __init__(self, addon, xbmc):
+        """Create a Language object for this addon."""
+        self.addon = addon
+        self.xbmc = xbmc
+
+    def get_string(self, key):
+        """Return the localized string for a key."""
+        if key in local_strings.__dict__.values():
+            get_localized_string = self.addon.getLocalizedString
+        elif key in kodi_strings.__dict__.values():
+            get_localized_string = self.xbmc.getLocalizedString
+        else:
+            # en_gb doesn't have it, error!
+            return "<{0}!>".format(key)
+        ls = get_localized_string(key)
+        if ls is not None:
+            return ls
+        else:
+            # no localization, return <key>
+            return "<{0}>".format(key)
+
+
+if __name__ == "__main__":
```

Comment:
looks like this is for local development only, so please remove it from you submission